app/frontend/components/pages/accounts/accountsDashboard.tsx

```diff
@@ -82,7 +82,7 @@ const AccountsDashboard = () => {
           <p>
             <b>
               This feature is not supported on other wallets yet. If you decide to move your funds
-              to account other then the first account, you will not see these funds in other wallets
+              to an account other than the first account, you will not see these funds in other wallets
```

Comment:
the fixes have caused the line length to exceed 100 characters, making `yarn eslint` to fail. Could you please fix it?

Reply:
Done.